docker-bake.hcl

```diff
@@ -378,6 +378,7 @@ target "virtual-osbuild-ci-cXs" {
                         "python3-pyyaml",
                         "python3-rpm-generators",
                         "python3-rpm-macros",
+                        "python3-tomli-w",
```

Comment:
`python3-tomli-w` isn't available for c9s.  We'll have to use `python3-toml` for c9s and `python3-tomli-w` for c10s.

Reply:
Done